data/server.spec.js

```diff
@@ -13,13 +13,10 @@ describe("GET /", () => {
   });
 
   // Verifies that the server message is "API is up and running..."
-  it("Should return proper body.message", async () => {
+  it("Should return proper body.message", async (done) => {
     const response = await request(server).get("/");
     expect(response.type).toMatch(/json/i);
+    done()
   });
```

Comment:
Ooooh, this is dope! Good job implementing this async feature of jest. I just learned something new 😁